--- restorecond-3.4/Makefile	2022-05-18 16:51:03.000000000 +0200
+++ restorecond-3.4/Makefile.no-systemd	2023-02-23 10:58:47.046360618 +0100
@@ -11,8 +11,7 @@
 SYSTEMDUSERUNITDIR ?= $(shell $(PKG_CONFIG) --variable=systemduserunitdir systemd)
 
 autostart_DATA = sealertauto.desktop
-INITDIR ?= /etc/rc.d/init.d
-SELINUXDIR = /etc/selinux
+INITDIR ?= /etc/rc.d
 
 GIO_CFLAGS = -DHAVE_DBUS $(shell $(PKG_CONFIG) --cflags gio-2.0)
 GIO_LIBS = $(shell $(PKG_CONFIG) --libs gio-2.0)
@@ -41,7 +40,7 @@
 		fi ; \
 	done
 	-mkdir -p $(DESTDIR)$(INITDIR)
-	install -m 755 restorecond.init $(DESTDIR)$(INITDIR)/restorecond
+	install -m 755 restorecond.init $(DESTDIR)$(INITDIR)/rc.restorecond
 	-mkdir -p $(DESTDIR)$(SELINUXDIR)
 	install -m 644 restorecond.conf $(DESTDIR)$(SELINUXDIR)/restorecond.conf
 	install -m 644 restorecond_user.conf $(DESTDIR)$(SELINUXDIR)/restorecond_user.conf
@@ -49,10 +48,6 @@
 	install -m 644 restorecond.desktop $(DESTDIR)$(AUTOSTARTDIR)/restorecond.desktop
 	-mkdir -p $(DESTDIR)$(DBUSSERVICEDIR)
 	install -m 644 org.selinux.Restorecond.service  $(DESTDIR)$(DBUSSERVICEDIR)/org.selinux.Restorecond.service
-	-mkdir -p $(DESTDIR)$(SYSTEMDSYSTEMUNITDIR)
-	install -m 644 restorecond.service $(DESTDIR)$(SYSTEMDSYSTEMUNITDIR)
-	-mkdir -p $(DESTDIR)$(SYSTEMDUSERUNITDIR)
-	install -m 644 restorecond_user.service $(DESTDIR)$(SYSTEMDUSERUNITDIR)
 relabel: install
 	/sbin/restorecon $(DESTDIR)$(SBINDIR)/restorecond 
 
